packages/components/src/badge/index.tsx

```diff
@@ -0,0 +1,15 @@
+export type BadgeProps = {
+	count: number;
+} & React.HTMLAttributes< HTMLSpanElement >;
+
+export const Badge: React.FC< BadgeProps > = ( {
```

Comment:
It was very simple to convert this component to TS and I thought it would be good to have a working example of TS + the storybook args feature.